Scripts/Runtime/Content/UnityContentManager.cs

```diff
@@ -53,12 +49,11 @@ private void PreventDestroyOnLoad()
             }
             else
             {
-                LogWarning(
-                    $"This {nameof(UnityContentManager)} may be destroyed on a scene change. ContentRoot: {ContentRoot.name}\n" +
-                    $"The {nameof(ContentRoot)} provided is not at the root of the hierarchy."
+                Logger.Warning(
+            $"This {nameof(UnityContentManager)} may be destroyed on a scene change. ContentRoot: {ContentRoot.name}"
+                    + $"\nThe {nameof(ContentRoot)} provided is not at the root of the hierarchy."
```

Comment:
Odd indenting here?

Reply:
Hah, yea. Looks like Rider's annotation threw me off.
![image](https://user-images.githubusercontent.com/1426795/182644186-fd81dae4-deab-4900-88a3-405231b69ed9.png)

Reply:
Yeah it's annoying when it does that. I have a hot key setup to auto format the indenting to circumvent this.